src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -52,26 +52,26 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
     public function applyToItem(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, array $identifiers, string $operationName = null)
     {
         if (null !== $operationName) {
-            $context = ['item_operation_name' => $operationName];
+            $options = ['item_operation_name' => $operationName];
         }
 
-        $this->joinRelations($queryBuilder, $resourceClass, $context ?? []);
+        $this->joinRelations($queryBuilder, $resourceClass, $options ?? []);
     }
 
     /**
      * Get metadata properties for a given resource class.
      *
      * @param string $resourceClass
-     * @param array  $context       the serialization context
+     * @param array  $options       PropertyMetadataFactory options, more specifically used by the SerializerPropertyMetadataFactory
      *
      * @return array
      */
-    private function getMetadataProperties(string $resourceClass, array $context = []) : array
+    private function getMetadataProperties(string $resourceClass, array $options = []) : array
```

Comment:
I'd like to see this useless function removed. It can just be a nested loop where it's called, like everywhere else.